### DISCUSSION TITLE
Trigger Edit Form with Multiple Items Selected from Search Results

### DISCUSSION THREAD

**Discussion by dontthroworanges** [microsoft-search/pnp-modern-search#3838] [Q&A]
Is it possible to trigger the Edit form for multiple selected items from the Search Results web part? I would love to be able to edit multiple items right from the Search Results. 

Edit - I'm quite poor with handlebars so I didn't want to spend a lot of time mucking about with it if not possible. 

Thanks!

**Comment by wobba** [comment]
There is no edit of items in these web parts.
